network-providers/transaction_watcher.md

```diff
@@ -0,0 +1,31 @@
+## TransactionWatcher
+
+TransactionWatcher allows one to continuously watch (monitor), by means of polling, the status of a given transaction.
```

Comment:
If we'll switch to the `Awaiter` naming, perhaps also rephrase this a bit, to say that `it allows one to await until a specific event (such as completion) occurs on a given transaction` - or something similar.

Reply:
Rephrasing is optional (it's though better perhaps to omit the word `continuously`).

Reply:
Done!